.github/actions/test-coverage/action.yml

```diff
@@ -10,9 +10,6 @@ name: test-coverage
 description: Test coverage tracking for mobile repo
 
 inputs:
-  github_token:
-    description: The token to use to download the coverage result
-    required: true
```

Comment:
github_token was being used to post commit msg.  But that's no longer needed since using a different action instead straight http request.